Sites/PR4/styles.css

```
@@ -0,0 +1,454 @@
+.clearfix::before,
+.clearfix::after {
+    content: ' ';
+    display: table;
+}
+
+.clearfix::after {
+    clear: both;
+}
+
+h6 {
+    color: white;
+}
+
+.alternate {
+    font-size: 36px;
+    font-family: "HelveticaNeue-Bold", "Helvetica Neue";
+}
+
+.transparent {
+    opacity: 0.59;
+}
+
+.transparentblack {
+    color: black;
+}
+
+.grey {
+    color: grey;
+    font-weight: normal;
+}
+
+img {
+    width: 250px;
+    margin: 40px;
+}
+
+.box {
+    margin: 10px;
+    width: 200px;
+    height: 50px;
+    background: black;
+    position: relative;
+    text-align: center;
+    opacity: 0.59;
+}
+
+.containerwhite {
+    width: 375px;
+    height: relative;
+    background: white;
+}
+
+.containerblue {
+    width: 375px;
+    height: 667px;
+    background: #DEEDFC
+}
+
+div.containerwhite {
+    padding-top: 20px;
+    padding-right: 10px;
+    padding-bottom: 0px;
```

Zero values don't need measurements:
```suggestion
    padding-bottom: 0;
```